rules/imports.js

```diff
@@ -195,33 +204,32 @@ module.exports = {
     // Prevent unassigned imports
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unassigned-import.md
     // importing for side effects is perfectly acceptable, if you need side effects.
-    'import/no-unassigned-import': 'off',
+    'import/no-unassigned-import': 'error',
 
     // Prevent importing the default as if it were named
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-named-default.md
     'import/no-named-default': 'error',
 
     // Reports if a module's default export is unnamed
     // https://github.com/benmosher/eslint-plugin-import/blob/d9b712ac7fd1fddc391f7b234827925c160d956f/docs/rules/no-anonymous-default-export.md
-    'import/no-anonymous-default-export': ['off', {
-      allowArray: false,
+    'import/no-anonymous-default-export': ['error', {
+      allowArray: true,
       allowArrowFunction: false,
       allowAnonymousClass: false,
       allowAnonymousFunction: false,
-      allowLiteral: false,
-      allowObject: false,
+      allowLiteral: true,
+      allowObject: true,
     }],
 
     // This rule enforces that all exports are declared at the bottom of the file.
     // https://github.com/benmosher/eslint-plugin-import/blob/98acd6afd04dcb6920b81330114e146dc8532ea4/docs/rules/exports-last.md
-    // TODO: enable?
-    'import/exports-last': 'off',
+    'import/exports-last': 'error',
 
     // Reports when named exports are not grouped together in a single export declaration
     // or when multiple assignments to CommonJS module.exports or exports object are present
     // in a single file.
     // https://github.com/benmosher/eslint-plugin-import/blob/44a038c06487964394b1e15b64f3bd34e5d40cde/docs/rules/group-exports.md
-    'import/group-exports': 'off',
+    'import/group-exports': 'error',
```

Comment:
Yeah but it's easier when navigating. If you need to add export a function then it will be grouped in last statement like

```
const a = ...
 function b() {}

export { a, b };
```